android/src/main/java/com/brentvatne/react/VideoManagerModule.java

```diff
@@ -0,0 +1,39 @@
+package com.brentvatne.react;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+
```

Comment:
why do you need a new module ? can't you do like presentFullScreen apis ?

Reply:
Actually there is no `VideoManager` module in android now. To maintain unity between platforms, it seems to me that this is a good solution. But lets keep open this discussion